### PR TITLE
02.GIT.Local.S.Surdo

### PR DESCRIPTION
**Pull request by Esserg**
Siarhei Surdo 02.Git.Local

### CODE REVIEW

File: Siarhei_Surdo/02.Git.Local/02.GIT.Local.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+cd it-study/
```
Comment (pluhin) [inline Siarhei_Surdo/02.Git.Local/02.GIT.Local.md]: missing git markup

File: Siarhei_Surdo/02.Git.Local/log_online [hunk 2]
```diff
@@ -0,0 +1,34 @@
+pi@u18sdevops1:~/it-study/02.Git.Local$ git checkout master
```
Comment (pluhin) [inline Siarhei_Surdo/02.Git.Local/log_online]: missing git markup

File: Siarhei_Surdo/02.Git.Local/02.GIT.Local.md (hunk 1 above)
Comment (Esserg) [reply Siarhei_Surdo/02.Git.Local/02.GIT.Local.md]: Add git markup.
Check please.

File: Siarhei_Surdo/02.Git.Local/log_online (hunk 2 above)
Comment (Esserg) [reply Siarhei_Surdo/02.Git.Local/log_online]: Add git markup. 
Change file extension for log_online 
Github don't see my git markup for file without extension.
Check please.

**Pull request merged**